tests/mocha/json_test.js

```diff
@@ -8,32 +8,22 @@ suite('JSON Block Definitions', function() {
   setup(function() {
     sharedTestSetup.call(this);
     this.workspace_ = new Blockly.Workspace();
-    this.blockTypes_ = [];
```

Comment:
In this file I think you should call Blockly.defineBlocksWithJsonArray directly, rather than through your helper. That's because in this file you're explicitly testing that function, while in the others you just need to assume your helper will work correctly and you don't need to worry about any extra work it's doing.

Reply:
I don't think I understand why not to use the helper function here.
If I don't use the helper, I'd still need logic to add all the block types to cleanup, which seems like I should just use the helper.

Reply:
Yeah, on reflection using the helper seems fine, as long as you know it's a thin wrapper.
My concern is around helpers that do extra work beyond that.